### PR TITLE
[Merged by Bors] - chore: deprecate MulEquiv.map_mul, AddEquiv.map_add

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by j-loreaux** [comment]
This doesn't actually include any deprecations. Is your plan to add them, or change the PR title?

File: Mathlib/Algebra/Group/Equiv/Basic.lean [hunk 1]
```diff
@@ -215,6 +215,9 @@ theorem coe_toMulHom {f : M ≃* N} : (f.toMulHom : M → N) = f := rfl
 protected theorem map_mul (f : M ≃* N) : ∀ x y, f (x * y) = f x * f y :=
   _root_.map_mul f
 
+attribute [deprecated map_mul (since := "2024-08-08")] MulEquiv.map_mul
+attribute [deprecated map_add (since := "2024-08-08")] AddEquiv.map_add
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Group/Equiv/Basic.lean:219]: Deprecations are here

Comment (j-loreaux) [reply Mathlib/Algebra/Group/Equiv/Basic.lean:219]: Oops, sorry!

**Comment by j-loreaux** [comment]
bors merge

**Comment by j-loreaux** [comment]
Actually, we should bench this first, if the bench is fine, please merge.

bors d+

**Comment by j-loreaux** [comment]
!bench

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**